### PR TITLE
pushpa/PDB, looking for patch in order to support PDB for clamav

### PR DESCRIPTION
**Pull request by pushpamahadeva**
**Why we need this changes?**
Looking for PDB (pod disruption budget) for clamav application, to avoid down time.

### CODE REVIEW

**Comment by pushpamahadeva** [comment]
@desaintmartin please take a look and review the PR.

File: charts/clamav/values.yaml [hunk 1]
```diff
@@ -88,3 +88,6 @@ hpa:
   # average memory usage per pod (100Mi-1Gi)
   memory:
   requests:
+
+podDisruptionBudget: {}
```
Comment (desaintmartin) [inline charts/clamav/values.yaml:92]: historically, this has been done more like:
```suggestion
podDisruptionBudget:
  enabled: false
  maxUnavailable: 1
```

(https://github.com/helm/charts/search?q=podDisruptionBudget)

File: charts/clamav/templates/poddisruptionbudget.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{{- if .Values.podDisruptionBudget -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "clamav.fullname" . }}
```
Comment (desaintmartin) [inline charts/clamav/templates/poddisruptionbudget.yaml:5]: Could you add the default labels (see the other resources) ?

**Review by desaintmartin — CHANGES_REQUESTED** [review]
Thanks!
Could you bump a minor version in Chart.yaml? According to semver, this is new feature.

File: charts/clamav/templates/poddisruptionbudget.yaml (hunk 2 above)
Comment (pushpamahadeva) [reply charts/clamav/templates/poddisruptionbudget.yaml:5]: yep.. will do all related changes.

**Comment by pushpamahadeva** [comment]
@desaintmartin could you please take a look
and let me know if you need further changes.

File: charts/clamav/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: An Open-Source antivirus engine for detecting trojans, viruses, malware & other malicious threats. Using Mailu docker image.
 name: clamav
-version: 1.3.2
+version: 1.3.3
```
Comment (desaintmartin) [inline charts/clamav/Chart.yaml:4]:
```suggestion
version: 1.4.0
```

File: charts/clamav/templates/poddisruptionbudget.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{{- if .Values.podDisruptionBudget.enabled -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "clamav.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "clamav.name" . }}
```
Comment (desaintmartin) [inline charts/clamav/templates/poddisruptionbudget.yaml:7]:
```suggestion
    app.kubernetes.io/name: {{ include "clamav.name" . }}
    helm.sh/chart: {{ include "clamav.chart" . }}
    app.kubernetes.io/instance: {{ .Release.Name }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
```

**Review by desaintmartin — CHANGES_REQUESTED** [review]

File: charts/clamav/templates/poddisruptionbudget.yaml (hunk 4 above)
Comment (pushpamahadeva) [reply charts/clamav/templates/poddisruptionbudget.yaml:7]: please take a look now.

File: charts/clamav/templates/poddisruptionbudget.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  minAvailable: {{ .Values.podDisruptionBudget.minAvailable }}
+  {{- end }}
+  {{- if .Values.podDisruptionBudget.maxUnavailable }}
+  maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
+  {{- end }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "clamav.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end -}}
```
Comment (desaintmartin) [inline charts/clamav/templates/poddisruptionbudget.yaml:22]: I think a new line is wanted by CI here.

Comment (pushpamahadeva) [reply charts/clamav/templates/poddisruptionbudget.yaml]: yep.. lint failed
updated with new line.

**Comment by pushpamahadeva** [comment]
@desaintmartin need to trigger lint for the changes.

**Comment by pushpamahadeva** [comment]
@desaintmartin checks got passed, could please take a look.

**Review by desaintmartin — APPROVED** [review]
Thanks for the contribution!

**Comment by pushpamahadeva** [comment]
Thanks for the support and help @desaintmartin .

**Pull request merged**